### ISSUE TITLE
Upgrade to Leaflet 0.8 (a.k.a ~1.0)

### ISSUE THREAD

**Issue by leplatrem** [GeotrekCE/Geotrek-admin#1265]
- Vector loading is 8x faster (see #217)
- It allows to choose layer panes, which would allow to get rid of costly and hazardeous code in Geotrek to some layers below or on top of others
- It allows mixing Canvas and SVG layers on the same map. It will be interesting to put Canvas everywhere the objects layers are marked as non clickable. For example, everywhere the paths layer is shown in background : https://github.com/makinacorpus/Geotrek/blob/v0.27.2/geotrek/core/static/core/main.js#L22-L31 or for zoning layers, etc.

**Comment by SBats** [comment]
Need confirmation from @gutard but the one in Django Leaflet I suppose. 
Anyway the version 0.8 / 1.0 is not out yet, (still in rc) and there're some breaking change that need to be tested prior to upgrade dependencies.

**Comment by camillemonchicourt** [comment]
OK thanks.

**Comment by leplatrem** [comment]
Exactly! 0.7.x. The upgrade is likely to require a tremendous amount of work, especially if they decide to switch lat/lon...

**Comment by camillemonchicourt** [comment]
1.0 is out now since a few months.

**Comment by camillemonchicourt** [comment]
Maybe check if https://github.com/robertleeplummerjr/Leaflet.glify (or similar ?) could not help with data performances
